File: leetcode/easy/3014-minimum-number-of-pushes-to-type-word-i/solution.py
class Solution:
    def minimumPushes(self, word: str) -> int:
        curpush = 1
        res = 0
        count = 0
        for c in word:
            if count > 0 and count % 8 == 0:
                curpush += 1
            res += curpush
            count += 1
        return res

File: leetcode/easy/3014-minimum-number-of-pushes-to-type-word-i/solution.java
class Solution {
    public int minimumPushes(String word) {
        int curpush =1;
        int res = 0;
        int count =0;
        for(char c:word.toCharArray()){
            if(count>0 && count%8==0){
                curpush++;
            }
            res += curpush;
            count++;
        }
        return res;
    }
}